plan/client.go

```diff
@@ -89,19 +73,7 @@ func (c Client) Update(id string, params *stripe.PlanParams) (*stripe.Plan, erro
 	if params != nil {
 		commonParams = &params.Params
 		body = &stripe.RequestValues{}
-
-		if len(params.Name) > 0 {
-			body.Add("name", params.Name)
-		}
-
-		if len(params.Statement) > 0 {
-			body.Add("statement_descriptor", params.Statement)
-		}
-
-		if params.TrialPeriod > 0 {
-			body.Add("trial_period_days", strconv.FormatUint(params.TrialPeriod, 10))
-		}
-
+		params.Params.AppendTo(body)
```

Comment:
This feels like a breaking change to me. If someone has been passing in a `params` object with the `Currency` field set, we'll now append that to the request and cause a 400 error. We may want to make this an explicit breaking change and have different types of `PlanParams` objects per request.